pkg/utils/test/oc/literal.go

```diff
@@ -0,0 +1,64 @@
+package oc
+
+import (
+	"fmt"
+	"strings"
+)
+
+// For oc commands not a part of this package, e.g. oc.Logs, oc.Apply, oc.Delete etc,
+// oc.Literal is a workaround to run those commands.
+
+// ILiteral is an interface for collecting the command string
+type ILiteral interface {
+	Command
+
+	// an oc command string
+	From(string, ...interface{}) ILiteral
+}
+
+type literal struct {
+	*runner
+
+	cmdstr string
+}
+
+// Literal takes an oc command string and runs it
+func Literal() ILiteral {
```

Comment:
How about 
```suggestion
func Literal(cmd string, args ...interface{)) ILiteral {
  l.cmdstring = "oc" + strings.Trimspace(....) 
  
```
So the call would be 
```
oc.Literal("apply foobar")